releasenotes/notes/fix-magnum-trusts-ee4824c8e6e3610d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    backport fixes for Magnum on Keystone RBAC failure.
+    That caused by unexpected parameter.
```

Comment:
This is again very unhelpful, this would make more sense:

Resolved regression causing Magnum clusters failing to create caused by the usage of new version of OpenStack SDK

Something along those lines, what you said reading that alone literally gives no context for the user.